solutions/bio2/KeepOnTheGrass/index.tsx

```diff
@@ -0,0 +1,83 @@
+import { SCode, SCodeBlock, HStack, SText, STitle, Image } from "components";
+import { brace } from "../../util";
+
+export const KeepOnTheGrass = () => {
+  return (
+    <>
+      <STitle>Observation 1</STitle>
+      <SText>
+        The pattern is symmetrical, so we can just solve
+        the problem for positive $X$ and $Y$. Therefore,
+        we can assume that $X, Y \geq 0$. Also, without
+        loss of generality, we can assume that $X \geq Y$,
+        because we can swap $X$ and $Y$ and the answer will
+        still be the same.
+      </SText>
+      <STitle>Observation 2</STitle>
+      <SText>
+        The diagrams given in the problem statement are
+        deliberately misleading, because they do not quite
+        give enough squares to show how the pattern proceeds.
+        However, by generating the next few "terms" of the
+        pattern, we can see that the pattern is actually
+        periodic - if we let $f(x, y)$ be the colour of
+        the square at coordinates $(x, y)$, then we have
+        $f(x, y) = f(x + 4n, y + 4m)$ for all integers $n, m$.
```

Comment:
diagram is wrong

Reply:
ouch... I guess the diagram given in the official BIO2 editorial turned out to be the misleading one. LOL